test_cases/widget_form_save.js

```diff
@@ -126,41 +126,49 @@ export default () => {
             .type("-1")
             .clear()
             .type("-185");
-         cy.get(
-            '[data-cy="button save 63c31c80-81ff-4114-8aec-cd4d43c2533a"]'
-         ).click();
+         cy.get('[data-cy="button save 63c31c80-81ff-4114-8aec-cd4d43c2533a"]')
+            .should("be.visible")
+            .click();
 
          // pre-setup the data in the popup
          cy.get(
             '[data-cy^="menu-item conditional record rule de0a9a23-8204-4317-9026-97ede2670f79"]'
          )
             .contains("Condition rule")
-            .should("exist")
+            .should("be.visible")
             .click();
-         cy.get('[data-cy^="connectObject Orders"]').click();
+
+         cy.get('[data-cy^="connectObject Orders"]')
+            .should("be.visible")
+            .click();
+
          cy.get('[data-cy^="connectObject options"]')
-            .filter(":visible")
-            .click({ force: true });
+            .should("be.visible")
+            .click();
+
          cy.get(
             '[data-cy^="detail text balance check example b42e1e16-15ff-4c85-abb5-4b2e9d80ff32"]'
          )
-            .filter(":visible")
+            .should("be.visible")
+            .click();
+
+         cy.get('[data-cy^="connectObject Processes"]')
+            .should("be.visible")
             .click();
 
-         cy.get('[data-cy^="connectObject Processes"]').click();
          cy.get('[data-cy^="connectObject options"]')
-            .filter(":visible")
-            .first()
```

Comment:
I think we still need `.first()` here. If I remember correctly the data for this test is a bit weird. If the test gets retried, multiple options will be here causing an error trying to click multiple.